controllers/object_controls.go

```diff
@@ -3365,6 +3530,21 @@ func transformValidationInitContainer(obj *appsv1.DaemonSet, config *gpuv1.Clust
 					setContainerEnv(&(obj.Spec.Template.Spec.InitContainers[i]), env.Name, env.Value)
 				}
 			}
+			setContainerEnv(&(obj.Spec.Template.Spec.InitContainers[i]), DriverManagedByOperatorEnvName, fmt.Sprintf("%v", config.Driver.IsEnabled()))
+
+			// Pass env if custom driver and/or dev roots are configured with the operator
+			if config.HostPaths.DriverInstallDir != "" && config.HostPaths.DriverInstallDir != DefaultDriverInstallDir {
+				setContainerEnv(&(obj.Spec.Template.Spec.InitContainers[i]), "DRIVER_ROOT", config.HostPaths.DriverInstallDir)
```

Comment:
Is `DRIVER_ROOT` the env that we're using? Should this be a constant?

Reply:
I have not moved this to a constant since we are now only setting this specific environment variable once in `transformForDriverInstallDir()`.